ru/turing/courses/lesson2/Person.java

```diff
@@ -0,0 +1,110 @@
+package ru.turing.courses.lesson2;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.Period;
+import java.time.format.DateTimeFormatter;
+import java.util.regex.Pattern;
+
+
+public class Person {
+    private String name, surname, secondName, address;
+    private LocalDate data;
+
+    public String getIm() {
```

Comment:
Геттеры и сеттеры должны называться в соответствии с названиями полей. Этот и остальные нужно исправить

```suggestion
    public String getName() {
```